src/Files.App/Dialogs/AddItemDialog.xaml

```diff
@@ -41,11 +45,15 @@
 				IsItemClickEnabled="True"
 				ItemClick="ListView_ItemClick"
 				ItemsSource="{x:Bind ViewModel.AddItemsList}">
+
 				<ListView.ItemTemplate>
 					<DataTemplate x:DataType="vm:AddItemDialogListItemViewModel">
 						<Grid Height="50">
+
 							<StackPanel VerticalAlignment="Center" Orientation="Horizontal">
+
 								<Grid Margin="0,0,10,0" VerticalAlignment="Center">
```

Comment:
Won't `48/52` (for the height) and `8/12` (for the margin) work instead?

Reply:
Good catch